Lib/test/libregrtest/refleak.py

```diff
@@ -62,9 +65,11 @@ def dash_R(ns, test_name, test_func):
     def get_pooled_int(value):
         return int_pool.setdefault(value, value)
 
-    nwarmup, ntracked, fname = ns.huntrleaks
-    fname = os.path.join(os_helper.SAVEDCWD, fname)
-    repcount = nwarmup + ntracked
+    warmups = hunt_refleak.warmups
```

Comment:
Looks like this has broken all of the refleak buildbots: https://buildbot.python.org/all/#/builders/474/builds/1335/steps/5/logs/stdio

Reply:
They are all failing in the same way on #109197

Reply:
Ah. A test is missing apparently. I wrote PR #109202 to fix the regression and add a test.